deploy/charts/rook-ceph/templates/clusterrole.yaml

```diff
@@ -380,7 +380,7 @@ metadata:
 rules:
   - apiGroups: [""]
     resources: ["nodes"]
-    verbs: ["get", "list", "update"]
+    verbs: ["get", "list", "watch"]
```

Comment:
Do we need list and watch permission?

Reply:
we have kept `get,list and watch` for the pods https://github.com/ceph/ceph-csi/blob/devel/deploy/rbd/kubernetes/csi-provisioner-rbac.yaml#L17 , infroduced for topology mainly..  So this keep it in parity.  however the RBAC in general for various objects can be revisited seperately too.

Reply:
@Madhu-1 ptal.. thanks..

Reply:
This is for cephfs, we dont need any node access for cephfs as it does not support topology. Am removing all the unwanted RBAC's here https://github.com/rook/rook/pull/10033. if we just want to remove one in this PR am good with that :+1:

Reply:
yeah, we can actually keep this PR in to be parity with current CSI version of the same.  As a follow up of complete revisit of the RBAC as you are doing in another PR can be considered based on some more testing...etc later. :+1: